### ISSUE TITLE
Update the Warpwire app URL when the app becomes 'available'

### ISSUE THREAD

**Issue by ewendland23** [DukeLearningInnovation/kits#279]
Warpwire has moved to a new, cloud-based version.  We need to test that Toolkits still integrates correctly.

- [x] warpwire-frontend-test.oit.duke.edu is now pointing to their cloud servers
- [ ] ~~shib authentication not fully passed to WW. Clicking the WW login button takes me to WW with netid login button. Clicking on button opens a new tab, authenticate me, close the extra tab, and refresh my main tab with me logged in with access to the correct library.~~
- [ ] ~~Clicking back after being taken to WW brought the 'resource being setup' snackbar and dimmed the Warpwire app in the kit as if it was being setup again both before and after I logged into WW.~~

**Comment by ewendland23** [comment]
I believe this is working correctly, but i never have any media.  Hoping someone from LI can test.

**Comment by profmikegreene** [comment]
Got this beauty when adding WW from kits-test not sure if that's @ewendland23, Todd, or WW's problem.

<img width="742" alt="https___warpwire-frontend-test_oit_duke_edu_api_" src="https://user-images.githubusercontent.com/3632822/48801494-81d44d00-ecda-11e8-82d6-6c6c1aabc82a.png">

**Comment by ewendland23** [comment]
Hi @profmikegreene,

Can you tell me what you were doing when that happened?  Which Kit, what did you click, etc?  Does it always happen when you did that or just once?  Can you make it happen again?

Thanks!
Liz

**Comment by profmikegreene** [comment]
Should be able to recreate by

1. add warpwire from app store
1. give name/visibility/etc, click save & add to kit
1. after resource setup is complete, click warpwire icon from My Kits

**Comment by profmikegreene** [comment]
New incognito window/shib session.

I think the first launch issue is caused by me not doing a page refresh before clicking WW which I believe @noahhh is already working on.

Second launch issue where I'm unauthenticated when first hitting warpwire is still an issue though but it at least takes me directly to the folder after authentication, not the WW home screen.

**Comment by hrvalli** [comment]
I'm doing some preliminary testing on Kits and tried to add Warpwire to a kit. It looked like it was coming in and I got the green message that it was being set up. But it's been five minutes, and I've reloaded the site. But it's still grayed out and I can't click on it.

**Comment by ewendland23** [comment]
Hi Heather,

Thanks for testing!  

Are you able to see this error now?  If so, can you tell me which Kit you are using?  I am not sure what application is giving the error.

And if you find bugs in the future, it is super helpful if you can provide more information - like which Kit you are working with, approximately when you saw the error (so i can look in the logs) and screen shots that include the URL (if possible).

Thanks again!
Liz

**Comment by hrvalli** [comment]
Hi Liz. Sorry, I'll try to include more useful data in the future. I was in "Crosslisted  Course F18" between 3:30-4:15 yesterday when this error occurred. 
<img width="336" alt="screen shot 2018-11-28 at 9 44 01 am" src="https://user-images.githubusercontent.com/19862293/49159360-58c34600-f2f2-11e8-9407-d6da27bf0723.png">

**Comment by ewendland23** [comment]
Ok, so this isn't happening now so I think this is related to issue where we aren't updating the app URL when the app becomes 'available'.  Noah knows about this and is fixing it.  If you reload the page and then click the app, it should work fine.

Thanks again Heather!

**Comment by profmikegreene** [comment]
Renamed since the whole discussion was focused on this particular issue Heather reported. The initial issue appears to be fixed in Firefox. 

All this in Firefox Mac.

However, I while I did not receive the 404 or plain HTML error reported, I was not fully authenticated to Warpwire even though I did go to the correct WW library for kit. Clicking the WW login button did open a new tab, authenticate me, close the extra tab, and refresh my main tab with me logged in with access to the correct library.

Clicking back also brought the 'resource being setup' snackbar and dimmed the Warpwire app in the kit as if it was being setup again both before and after I logged into WW.
